sound/soc/sof/trace.c

```diff
@@ -154,16 +163,24 @@ int snd_sof_init_trace_ipc(struct snd_sof_dev *sdev)
 	if (sdev->dtrace_is_enabled)
 		return -EINVAL;
 
+	/* update init_offset, and reinitial offset to 0 */
+	sdev->trace_init_offset = (sdev->trace_init_offset +
+				  sdev->trace_offset) %
+				 sdev->dmatb.bytes;
```

Comment:
@keyonjie is this correct? should this just be 
sdev->trace_init_offset = sdev->trace_offset % sdev->dmatb.bytes?

trace_init_offset never gets reset so I am concerned this might skew the offset.

Reply:
no, trace_init_offset will be refreshed at each resume.
trace_offset is the one come from FW directly.

Reply:
@keyonjie yes, this makes sense. Sorry about the confusion